sklearn/feature_extraction/text.py

```diff
@@ -996,8 +996,8 @@ class CountVectorizer(_VectorizerMixin, BaseEstimator):
         will be removed from the resulting tokens.
         Only applies if ``analyzer == 'word'``.
 
-        If None, no stop words will be used. max_df can be set to a value
-        in the range [0.7, 1.0) to automatically detect and filter stop
+        If None, no stop words will be used. ``max_df`` can be set to a value
+        in the range [0.0, 1.0] to automatically detect and filter stop
         words based on intra corpus document frequency of terms.
```

Comment:
In the context of `stop_words`, recommending a higher `max_df` means that words that appear more will be ignored, which is frequently the case for stop words.

With that in mind, I think we can clarify this statement:

```suggestion
        If None, no stop words will be used. In this case, setting `max_df`
        to a higher value, such as in the range (0.7, 1.0), can automatically detect
        and filter stop words based on intra corpus document frequency of terms.
```